go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	go.uber.org/zap v1.24.0
 	golang.org/x/net v0.0.0-20220805013720-a33c5aa5df48
 	golang.org/x/sys v0.3.0
+	gonum.org/v1/gonum v0.12.0
```

Comment:
Are we ok this this dependency? I never heard of it, so curios if it is a super used library...

Reply:
It's part of a [larger suite of mathematical packages](https://www.gonum.org/) which seems to be quite professionally maintained, and appears to have a pretty good number of users (6.2k stars).